### PR TITLE
Refactor components, added Ben's Profile components.

### LINKED ISSUE #18: Debounce Issue when spamming second Login button
Can open multiple screens of the home pages (where navigation bar is).

### PR DESCRIPTION
**Pull request by exelarios**
- Fixed Button Debounce
- Refactored random components
- Implemented a custom TextInput component instead of built-in.
- Implemented & Refactored Ben's Profile and user settings components.

### CODE REVIEW

File: screens/Contact/Contact.component.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+    <View>
+        <TextLabel style={styles.input} multiline={true} placeholder="To:" />
+        <TextLabel
+        style={styles.input}
+        multiline={true}
+        textStyle={{minHeight: 464}}
+        />
+        <Button label="Submit" onPress={() => console.log("yes")}/>
+    </View>
```
Comment (lchua2314) [inline screens/Contact/Contact.component.js:26]: Nitpick: I would like an indenting for View tag, tag's content and closing tag.

File: screens/Contact/Contact.styles.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: '#00FA9A',
+    },
+    input: {
+        marginTop: 0,
+        marginLeft: 10,
+        marginRight: 10,
+        backgroundColor: '#FFFFFF',
```
Comment (lchua2314) [inline screens/Contact/Contact.styles.js]: Max white color (and black) is discouraged since it could strain the eyes. Maybe try "#FAFAFA" or something.

Comment (lchua2314) [reply screens/Contact/Contact.styles.js]: Might have to change all pure white (and black) colors too in other files too.

File: screens/UserSettings/UserSettings.component.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import styles from './UserSettings.styles';
+
+const UserSettings = () => {
+    return(
+        <View style={styles.spacing}>
+           <TextLabel label="Account Name:" placeholde="John Doe"/>
+           <TextLabel label="Current Password:" placeholde="Password"/>
+           <TextLabel label="New Password:" placeholde="Password"/>
+           <TextLabel label="Confirm Password:" placeholde="New Password"/>
```
Comment (lchua2314) [inline screens/UserSettings/UserSettings.component.js]: The above four tags say "placeholde" for an attribute name.

**Review by lchua2314 — APPROVED** [review]
Left a few comments on the File Changes. Please make changes before merge. Other than that, it looks good.

**Review by peterc3989 — APPROVED** [review]

**Pull request merged**